drgn_tools/scsiHosts.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+"""
+Helper to print scsi hosts
+"""
+from typing import List
+
+from drgn import container_of
+from drgn import Object
+from drgn import Program
+from drgn.helpers.linux.list import list_for_each_entry
+
+
+def _list_of_scsi_host(prog: Program) -> List[Object]:
+    """
+    Iterates through all scsi hosts and returns a
+    list of scsi devices
+    """
```

Comment:
This is marked private (starts with an `_`), but it looks useful. Could you name it such that it's not private? And also, be sure to document the C type of the returned objects in the docstring. EG:

```suggestion
def for_each_scsi_host(prog: Program) -> List[Object]:
    """
    Iterates through all scsi hosts and returns a
    list of scsi devices
    :returns: list of ``struct Scsi_Host *``
    """
```